blocks/quilted-image-block/features/quilted-image/default.jsx

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useContent } from "fusion:content";
+import { useFusionContext } from "fusion:context";
+
+// Arc Themes Components - Base set of components used to compose blocks
+// https://github.com/WPMedia/arc-themes-components/
+import {
+	Heading,
+	HeadingSection,
+	Image,
+	Button,
+	Paragraph,
+	Stack,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-quilted-image";
+function QuiltedImage({ customFields }) {
+	const {
+		headline,
+		image1URL,
+		overlay1Text,
+		overlay1TextVariant = "dark",
+		button1Text,
+		// NOTE : Link href properties currently need a default or "startsWith of undefined" error is raised.
+		// https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.00/src/components/link/index.jsx#L22
+		button1Action = "",
+		button1Variant = "dark",
+		image2URL,
+		overlay2Text,
+		overlay2TextVariant = "dark",
+		button2Text,
+		button2Action = "",
+		button2Variant = "dark",
+		image3URL,
+		overlay3Text,
+		overlay3TextVariant = "dark",
+		button3Text,
+		button3Action = "",
+		button3Variant = "dark",
+	} = customFields;
+
+	// Translate user-facing variant name into values component is expecting.
+	const getButtonVariant = (value) => {
+		return value === "dark" ? "primary" : "secondary";
+	};
+
+	// get properties from context for using translations in intl.json
+	// See document for more info https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Lokalise+and+Theme+Blocks
+	// const { arcSite } = useFusionContext();
+	// const { locale } = getProperties(arcSite);
+	// const phrases = getTranslatedPhrases(locale);
+
+	// Account for "dark" or "light" variants y
+	/*
+  const cssClasses = [
+		BLOCK_CLASS_NAME,
+		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
+	].join(" ");
+  */
```

Comment:
are there internationalization aspects to this ticket? it looks like you might not need translations here?

Reply:
removed

Reply:
removed